fastcrypto/src/tests/signature_tests.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::{rngs::StdRng, SeedableRng};
+
+use crate::{
+    hash::{HashFunction, Sha256},
+    traits::KeyPair,
+};
+
+/// Generate `n` random key pairs.
+pub fn keys<KP: KeyPair>(n: usize) -> Vec<KP> {
```

Comment:
ah, these are util functions, not tests, so we could rename the file to signature_test_utils.rs